packages/grafana-ui/src/themes/_variables.scss.tmpl.ts

```diff
@@ -198,6 +198,11 @@ $panel-header-z-index: 10;
 // tabs
 $tabs-padding: 10px 15px 9px;
 
+// fusebit
+$fusebit-panel-bg: rgb(255, 255, 255);
+$fusebit-text-color: #333333;
+$fusebit-font: 'Courier Prime', monospace;
+
```

Comment:
Shouldn't this come from the theme?

Reply:
They are used on .scss files, so we have to use .scss variables